OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/PermissionsForRoleControllerTests.cs

```diff
@@ -27,13 +28,14 @@ public class PermissionsForRoleControllerTests
 
     private IEnumerable<PermissionsForRole> permissionsForAllRoles;
     private PermissionsForRole permissionsForRoleEntity;
-
-
+    private IMapper mapper;
 
     [SetUp]
     public void Setup()
     {
         service = new Mock<IPermissionsForRoleService>();
+        var config = new MapperConfiguration(cfg => cfg.AddProfile<Util.MappingProfile>());
+        mapper = config.CreateMapper();
```

Comment:
` var config = new MapperConfiguration(cfg => cfg.AddProfile<Util.MappingProfile>());
        mapper = config.CreateMapper();`
looks like a method in TestsHelper